.github/workflows/ci.yml

```diff
@@ -625,5 +625,5 @@ jobs:
         with:
           # Name prefix is checked in the `Annotate checks` workflow
           name: test report pt (${{ matrix.config }}, ${{ matrix.suite }}, ${{ matrix.jdk }})
-          path: testing/trino-product-tests/target/reports/testng-results.xml
+          path: testing/trino-product-tests/target/reports/**/testng-results.xml
```

Comment:
Can you point out some tests where the testng results aren't present under `reports` dir?

Or is the issue that some product tests don't produce reports at all?
What happens when a PT env fails to start at-all? Or when the PT itself doesn't start and get skipped/failed?

Reply:
If the pt fails, this step will be skipped. If the pt won't create any report, there'll be a warning annotation, like now, because this path is invalid. See the screen shot in the PR description

Reply:
So is my assumption that this change is needed because some PTs write reports under a subdirectory correct? Or was the path wrong all along - and all PTs write reports under a subdirectory?

Sorry if I'm being dense but it's unclear to me.

Reply:
The path was wrong all along, there's always a subdir with a UUID as its name. I added a recursive wildcard to handle all cases if that's to change.

No problem, I don't mind explaining something that should have been in the PR description :-)

Reply:
Please add this to the commit message and we are good to go.

Reply:
Done!